dm_control/locomotion/walkers/assets/dog_v2/add_wrapping_geoms.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2023 The dm_control Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or  implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+"""Add wrapping geometries to the dog model, needed for the muscle actuators"""
+
+
+def add_wrapping_geoms(model):
+    pelvis = model.find('body', 'pelvis')
+    pelvis.add('geom', name='hip_R_wrapping', dclass='wrapping',
+        size=[0.05, 0.04], pos=[0, 0.05, -0.02])
+    pelvis.add('site', name='hip_R_wrapping_forward',
+        dclass='wrapping', pos=[0.08, -0.05, -0.00])
+    pelvis.add('site', name='hip_R_wrapping_backward',
+        dclass='wrapping', pos=[-0.04, -0.05, -0.04])
+
+    pelvis.add('geom', name='hip_L_wrapping', dclass='wrapping',
+        size=[0.05, 0.04], pos=[0, -0.05, -0.02])
+    pelvis.add('site', name='hip_L_wrapping_forward',
+        dclass='wrapping', pos=[0.08, 0.05, -0.00])
+    pelvis.add('site', name='hip_L_wrapping_backward',
+        dclass='wrapping', pos=[-0.04, 0.05, -0.04])
+
+    C_2 = model.find('body', 'C_2')
+    C_2.add('geom', name='C_2_wrapping', dclass='wrapping',
+        pos=[0.02, 0, 0.03], size=[0.015, 0.04])
+    C_2.add('site', name='C_2_wrapping_forward', pos=[0.05, 0, 0.02], dclass='wrapping')
+    C_2.add('site', name='C_2_wrapping_backward', pos=[-0.005, 0, 0.04], dclass='wrapping')
+    C_7 = model.find('body', 'C_7')
+    C_7.add('geom', name='C_7_wrapping', dclass='wrapping', size=[0.03, 0.07])
+    Ca_1 = model.find('body', 'Ca_1')
+    Ca_1.add('geom', name='Ca_1_wrapping', size=[0.008, 0.04], dclass='wrapping')
+    Ca_1.add('site', name='Ca_1_wrapping_forward', pos=[0, 0, -0.05], dclass='wrapping')
+    Ca_1.add('site', name='Ca_1_wrapping_backward', pos=[0, 0, 0.05], dclass='wrapping')
+
+    # L_1 = model.find('body', 'L_1')
```

Comment:
Either delete the commented code or add an option to activate it that is off by defaults